packages/config/src/store/loadStoreConfig.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
not blocking but I would prefer to not use whole-file disables

we should either 1) decide to disable this globally or 2) ignore individual lines/usages